src/app.ts

```diff
@@ -59,7 +59,7 @@ import { XMLHttpRequest } from 'xmlhttprequest-ts';
 import { uptime } from './lib/tools/time';
 
 ON_DEATH({ uncaughtException: true })((signalOrErr, origin) => {
-    const crashed = signalOrErr !== 'SIGINT';
+    const crashed = ['SIGINT', 'SIGTERM'].indexOf(signalOrErr as 'SIGINT' | 'SIGTERM' | 'SIGQUIT') === -1;
```

Comment:
you probably should use `includes` instead of `indexOf`

Reply:
Yeah, I have that thought too. Will change it. Thanks.